docs/site/todo-tutorial-controller.md

```diff
@@ -46,7 +46,7 @@ repository, which we'll use to perform our operations against the datasource.
 
 ```ts
 import {repository} from '@loopback/repository';
-import {TodoRepository} from '../repositories';
+import {TodoRepository} from '../repositories/todo.repository';
```

Comment:
I think we can keep this line unchanged.  There's `src/repositories/index.ts` that export the todo.repository.
https://github.com/strongloop/loopback-next/blob/master/examples/todo/src/repositories/index.ts